app/views/api/backend_usages/index.html.slim

```diff
@@ -1,11 +1,15 @@
-h2 Backends
+h1 Backends
+p.intro For a product to work, it needs to have at least one backend with a Private Base URL (your API). If multiple backends are added to a product, each backend will have a unique public path.
```

Comment:
You mentioned in https://github.com/3scale/porta/pull/1710/files#r381168826 that Docs team don't want us to use capitals in that case... should it be consistent here as well?

Reply:
It applies to the rest of the changes where the occurrence of `Private Base URL` is found

Reply:
yeah, you're right.